examples/calculator/pom.xml

```diff
@@ -6,7 +6,7 @@
     <description>A simple calculator application</description>
     <groupId>com.spotify</groupId>
     <artifactId>calculator-service</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
But currently they're separate, and not built, so there'll be no problems with release:prepare either. I'll go with 0.0.1-SNAPSHOT (for both apollo-example (which we may remove/move to an example?) and example/calculator).